src/Columns/IColumn.cpp

```diff
@@ -540,6 +540,19 @@ void IColumnHelper<Derived, Parent>::collectSerializedValueSizes(PaddedPODArray<
     }
 }
 
+template <typename Derived, typename Parent>
+IColumnInsertFromFunc IColumnHelper<Derived, Parent>::getInsertFromFunc() const
```

Comment:
I suggest not making it part of the `IColumn` interface. Let's create a wrapper for column objects that will have non-virtual `insertFrom` and basically do the same trick under the hood. Besides not "polluting" the interface, it'd be probably easier to integrate it in other places in the code. 
WDYT?